### ISSUE TITLE
Oracle: Planetside Opportunity should be `Select table by: Life` not `...by: Region`?

### ISSUE THREAD

**Issue by farrago** [rsek/dataforged#37]
At https://github.com/rsek/dataforged/blob/main/_data-master/oracles/planet/planet.yaml#L134 I think it should be `Select table by: Life` not `...by: Region`

The context is:

```yaml
    - Name: Opportunity
      Display name: Planetside Opportunity
      Content tags: [opportunity]
      Repeatable: true
      Description: |-
        Choose or roll on this table when you want inspiration for a beneficial encounter or event on a planetside journey, such as when you roll a strong hit with a match as you Undertake an Expedition, or if you Explore a Waypoint and find an opportunity.
      Part of speech: [fragment]
      Select table by: Region
      Tables:
        - Display name: Planetside Opportunity - Lifebearing
          Name: Lifebearing
          Requires:
            Location: [Planetside]
            Life: [Lifebearing, Simple, Sparse, Diverse, Bountiful, Overrun]
          Table:
           ...
```
The sub-tables have `Requires` of `Location` and `Life`.  As Location is somewhat of a given, then it seems like it should be `Life` is selector?  This would also match with the formatting of the table on page 317 of the 122421 version of the backer preview pdf.

I would make a pull request to fix it, but I'm not quite sure of which files are canonical and which are built by tools.

Thanks for the amazing work on Dataforged in general!

**Comment by rsek** [comment]
ah, thanks for the catch - i suspect i got sloppy with a regex somewhere.

the YAML files are the canonical ones, but i generally recommend against making PRs [for various reasons](https://github.com/rsek/dataforged/issues/33#issuecomment-970232038). i'll add this fix to the incoming content update (the data entry of which is complete, i'm just in the midst of rewriting some of my build scripts so it might be a day or two).

**Comment by farrago** [comment]
No problem, and it's frankly amazing with the massive amount of content that there aren't more issues. Thanks for the hard work to make it so good!

**Comment by rsek** [comment]
resolved in ec476aa29a4deaa0be6a128b4e7fa46074d1f306
